ports_test.go

```diff
@@ -424,8 +425,75 @@ func testL2L3Convert(t *testing.T, plan string) {
 		t.Fatal(err)
 	}
 
-	if nType != NetworkL3 {
+	if nType != "layer3" {
 		t.Fatal("the device now should be back in network type L3")
 	}
 
 }
+
+func dev2nt(t *testing.T, c *Client, did, nt string) {
```

Comment:
maybe the func name here should be more descriptive? or at least add a comment to briefly mention this is taking a device id and setting it's network type. maybe also make the params a bit more verbose, `did` => `deviceID` `nt` => `networkType`?

Reply:
Yes, I will make it more descriptive. Thanks for review!